app/move/fields/create.js

```diff
@@ -242,7 +242,7 @@ module.exports = {
   to_location_court_appearance: toLocationType('court_appearance', {
     validate: 'required',
   }),
-  to_location_prison_recall: toLocationType('prison_recall', {
+  to_location_prison: toLocationType('prison_recall', {
```

Comment:
The first argument needs to match  the property name: `to_location_prison: toLocationType('prison', {`. Otherwise the field won't be processed correctly.